src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -184,7 +184,6 @@
     <RuntimeGroup Include="freebsd">
       <Parent>unix</Parent>
       <Architectures>x64;arm64</Architectures>
```

Comment:
FreeBSD 13.0 and previous are EOL 2021-04-13 (https://www.freebsd.org/releases/)

Reply:
If everything works on FreeBSD without these versions (and there is no version-specific nuget package), that would be great! but I don't think we are there yet.

FreeBSD 12.3, 12.4 and 13.1 are active versions and 12.2, 13.0 etc. are EOL. We should keep 12 and 13 in its list because on FreeBSD we only care about the major version.

cc @Thefrank

Reply:
Thank you for checking, I had misread the page.